pkg/v1/tkg/client/delete_region.go

```diff
@@ -470,15 +470,11 @@ func (c *TkgClient) waitForAVIResourceCleanup(regionalClusterClient clusterclien
 		if apierrors.IsNotFound(err) {
 			return nil
 		}
-		return errors.Wrapf(err, "unable to get ako statefulset")
```

Comment:
why should we remove this line? if statefulset fails to be get

Reply:
It is possible that once we set the `deleteConfig` to be true, the AVI resources have been cleaned up immediately. 
If we also use AVI to provide control plane HA, in that case, we may not get the ako statefulset using the client because of network issue.

Reply:
We should log a warning for easier debugging